test/validation-error-handling.test.js

```diff
@@ -64,6 +66,105 @@ test('should fail immediately with invalid payload', t => {
   })
 })
 
+test('should return custom error messages with ajv-errors', t => {
```

Comment:
Could you move these tests to [schema-examples.test.js](https://github.com/fastify/fastify/blob/master/test/schema-examples.test.js)

All the examples in Validation-and-Serialization are there